src/main/java/com/internet/shop/controller/user/RegistrationController.java

```diff
@@ -36,7 +38,12 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
         String repeatedPass = req.getParameter("repeatedPass");
         if (name.length() > 1 && login.length() > 3
                 && pass.length() > 5 && pass.equals(repeatedPass)) {
-            User newUser = new User(name, login, pass);
+            User newUser;
+            if (login.toLowerCase().contains("admin")) {
+                newUser = new User(name, login, pass, Set.of(Role.of("ADMIN")));
+            } else {
+                newUser = new User(name, login, pass, Set.of(Role.of("USER")));
+            }
```

Comment:
admin should not have a shopping cart
let's set only USER role to those users that register in our app

Reply:
Removed